inc/native/namespace.php

```diff
@@ -8,9 +8,6 @@
 use Altis;
 use Altis\Analytics\Experiments;
```

Comment:
Can remove this one now as well, looks like it was only used in the function you removed.

Reply:
Well spotted. Thanks